docs/Using Randomness/v2-subscription/vrf-v2-subscription.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: ' Subscription Method'
+permalink: 'docs/vrf/v2/subscription/'
+whatsnext:
+  {
+    'Get a Random Number': '/docs/vrf/v2/subscription/get-a-random-number/',
+    'Configuration': '/docs/vrf/v2/subscription/configuration/',
+  }
+metadata:
+  title: 'Generate Random Numbers for Smart Contracts using Chainlink VRF v2 - Subscription method'
+  description: 'Learn how to securely generate random numbers for your smart contract with Chainlink VRF v2(an RNG). This guide uses the subscription method.'
+---
+
+> ℹ️ You are viewing the VRF v2 guide.
+>
+> If you are using v1, see the [VRF v1 guide](/docs/chainlink-vrf/v1/).
+
+**Table of contents**
+
+- [Overview](#overview)
```

Comment:
```suggestion
```

I think we can remove the overview here since overview is implied at the top of the page.